tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -115,6 +115,13 @@ Search and Verify GPU Items Appears In Resources Page
     Search Items In Resources Section   GPU
     Check GPU Resources
 
+Verify Quick Starts Work As Expected
+    [Tags]  ODS-1166    ODS-1306
+    ...     ODS-1307    ODS-1308
+    Verify Quick Starts Work As Expected When All the Steps Marked As Yes   using-starburst-galaxy
```

Comment:
have you tried running this check for all the qs? For example yday you told me that some QuickStarts have something different from the usual structure (the seldon ones). in this way, these deviations cannot be captured

Reply:
As of now I have assumed that for every quickstart I will get **Check your work** box.

Reply:
hm I think it would be better to avoid this assumption fot the final version of this PR, since we know the assumption is not be valid in the reality

Reply:
I am using **Run Keyword And Continue On Failure** so if **Check your box** isn't available still it continues with the process.